FIPS/pom.xml

```diff
@@ -958,6 +958,46 @@
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
Joyce is working on to make non FIPS and FIPS as much same as possible and also doing common pom.xml, so all changes come in both products. Her PR is ready. Please check with her. We were waiting for the JDBC release, which is done yesterday, so we can merge her PR first.